longevity_test.py

```diff
@@ -368,6 +366,10 @@ def chunks(_list, chunk_size):
                 batch_params['stress_cmd'] += [params['stress_cmd']]
 
             self._run_all_stress_cmds(stress_queue, params=batch_params)
+
+            # Start nemesis thread after stress
+            self.db_cluster.start_nemesis()
```

Comment:
conceptually It's wrong to start the nemesis here.
The nemesis is part of the "test" and not part of the helper that it's intention is to run_user_stress_in_batches.
When adding the nemesis here like this, you prevent other uses of this helper for other cases.

Reply:
An option to solve it is to separate the "run_all" and the "verify_stress" to two different helpers and then call the nemesis in between.

Reply:
@roydahan I am not sure I understand your comment.  "run_all" and the "verify_stress" are two different helpers and I call the nemesis between them. Can you explain, please?

Reply:
You're running the nemesis inside a helper that is called "_run_user_stress_in_batches".
It's not related to nemesis and should be left without any relation to nemesis.

Reply:
@roydahan Changed. Please, review